docs/cloud/netdata-functions.md

```diff
@@ -43,36 +31,20 @@ functions - [plugins.d](https://github.com/netdata/netdata/blob/master/src/colle
 | Systemd-services   | System resource utilization for all running systemd services: CPU, memory, and disk IO.                                                                        | `systemd-cgtop`                 | no            | [cgroups](https://github.com/netdata/netdata/tree/master/src/collectors/cgroups.plugin#readme)                 |
 | Streaming          | Comprehensive overview of all Netdata children instances, offering detailed information about their status, replication completion time, and many more.        |                                 | yes           |                                                                                                            |
```

Comment:
```suggestion
| Streaming          | Comprehensive overview of all Netdata children instances, offering detailed information about their status, replication completion time, and many more.        |                                 | yes           |                                                                                                            |
| Netdata-api-calls | Real-time tracing of API calls made to the Netdata Agent. It provides information on query, source, status, elapsed time, and more. |                               | no |   |
```

Reply:
@Ancairon also not sure if `Streaming` function needs Cloud. have you confirmed it?

Reply:
I think @ilyam8 might know, I am not sure myself